src/components/PostPageDetail/index.js

```diff
@@ -684,7 +684,7 @@ const PostPageDetailIdComponent = (props) => {
   );
 };
 
-export default withInteractionsManaged(React.memo(PostPageDetailIdComponent));
+export default PostPageDetailIdComponent;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing `withInteractionsManaged` and `React.memo` may have implications on the performance of your component. `withInteractionsManaged` helps manage interactions and animations smoothly, while `React.memo` prevents unnecessary re-renders when props do not change. If these were removed for a valid reason, then it's fine. Otherwise, consider adding them back.
```diff
- export default PostPageDetailIdComponent;
+ export default withInteractionsManaged(React.memo(PostPageDetailIdComponent));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->